.release-please-manifest.json

```diff
@@ -0,0 +1,42 @@
+{
+  "detectors/node/opentelemetry-resource-detector-alibaba-cloud": "0.25.0",
+  "detectors/node/opentelemetry-resource-detector-aws": "0.25.0",
+  "detectors/node/opentelemetry-resource-detector-gcp": "0.25.0",
+  "detectors/node/opentelemetry-resource-detector-github": "0.25.0",
+  "metapackages/auto-instrumentations-node": "0.25.0",
+  "metapackages/auto-instrumentations-web": "0.25.0",
+  "packages/opentelemetry-browser-extension-autoinjection": "0.25.0",
```

Comment:
The manifest will be regenerated anyway on the next release so having extra entries is actually not a problem. Missing entries would be a bigger deal.